website/content/en/docs/administration/management.md

```diff
@@ -218,6 +218,13 @@ As you can see above, many administrative interfaces for Vector enable you to tr
 
 You can make Vector automatically reload itself when its [configuration file][configuration] changes by setting the `--watch-config` or `-w` [flag][watch_config] when you first start your Vector instance.
 
+Additionally you can add method for watching config change by setting the `--watch-config-method` to `recommended` or `poll`.
+
+`recommended` is default and it uses file event listener for file change events.
+`poll` can used where the event listener won't work. eg., attaching config files by NFS/EFS. which polls for file changes on certain interval.
```

Comment:
```suggestion
`poll` can used where the event listener won't work, eg. attaching config files by NFS/EFS, which will poll for file changes on certain interval.
```

Rephrasing this a little bit? maybe we can get a little help about technical writing in this file? @michaelcretzman